src/components/singlEventPageComponents/EditEvent.jsx

```diff
@@ -1,13 +1,30 @@
 import { doc, updateDoc } from "firebase/firestore";
 import { useRouter } from "next/router";
-import React from "react";
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import { MultiSelect } from "react-multi-select-component";
+
+import "react-datepicker/dist/react-datepicker.css";
 
 import { firestore } from "@/lib/firebase/controller";
 
+import LocationInput from "../reusableComponents/LocationInput";
+import { interestList } from "../../lib/interestsList";
+
 function EditEvent() {
+    const [startDate, setStartDate] = useState(new Date());
+    const [selectedInterets, setSelectedInterets] = useState([]);
+    const [loca, setLoca] = useState(null);
     const router = useRouter();
     const { id } = router.query;
```

Comment:
if this is edit the event i think we'll need to state for all inputs to fetch and edit them

Reply:
Yes, 100%